src/test/groovy/org/gradle/profiler/AbstractProfilerIntegrationTest.groovy

```diff
@@ -16,7 +16,7 @@ abstract class AbstractProfilerIntegrationTest extends Specification {
         "3.3", "3.4.1", "3.5",
         "4.0", "4.1", "4.2.1", "4.7",
         "5.2.1", "5.5.1", "5.6.3",
-        "6.0.1", "6.1", "6.6-rc-2"
+        "6.0.1", "6.1", "6.6"
```

Comment:
`6.6.1`?

Reply:
Done. I also upgradled the build to 6.6.1.